cobalt/cobalt.cc

```diff
@@ -39,15 +39,15 @@ int main(int argc, const char** argv) {
   // parameters, also update the my_argc below.
   static const char* my_argv[] = {
       argv[0], "--disable-fre", "--no-first-run", "--kiosk",
-      "--force-video-overlays",
+      "--force-video-overlays", "--single-process",
       // Enable remote devtools access.
       "--remote-debugging-port=9222",
       "--remote-allow-origins=http://localhost:9222",
       // This flag is added specifically for m114 and should be removed after
```

Comment:
Perhaps a TODO + bug here ?

Reply:
What is there TODO here that is not already logged as such?